lib/home/library/video_library.dart

```diff
@@ -109,6 +209,12 @@ class _VideoLibraryState extends State<VideoLibrary> {
 
   Future<void> fetchDocuments() async {
     final QuerySnapshot snapshot = await Firestore.streams.get();
+    final userSnapshot =
+        await Firestore.users.doc('test_participant').collection('streams').get();
```

Comment:
Instead of using `'test_participant'` here, it might be nice to show the favorites for whoever's currently logged in.